Fix target type value when changing Format as select

simPod opened the pull request:
Fixes #16

andig commented:
From looking at the changes (not testing) I still don't get what the actual fix is. I'm wondering:

- are we still sending timeseries requests per default?
- are we sending table requests per default for table panels?
- what happens when a query does not return the metric type?

Sorry if these are obvious- or I'll just have to try these cases ;)

simPod commented:
Ad the default: yes, it was specified on 2 places so I removed one

The fix is that there was property `format` that was being modified by the select. But the targets use property `type`. `format` was redundant so I removed it

Other two questions: I must check, don't know

andig commented:
> Other two questions: I must check, don't know

Could I ask you to check? Then I could update the release PR with this change included

simPod commented:
> are we sending table requests per default for table panels?

good catch, sending fix soon

> what happens when a query does not return the metric type?

why should it return a metric type? I see that it is in documentation for `table` requests but I guess it is not used anywhere. Am I wrong?

andig commented:
> why should it return a metric type? I see that it is in documentation for table requests but I guess it is not used anywhere. Am I wrong?

II was assuming we should return:

  - metric type as per response OR
  - type `table` if table panel OR
  - type `time series`

this seemed to be the original behaviour as I understood it?

simPod commented:
I guess but it seems to be redundant to return it as it's already contained in request. I haven't found any issue not returning it

simPod commented:
@andig WDYT? IMO it's ok when dropped
